src/planscape/metrics/models.py

```diff
@@ -0,0 +1,160 @@
+    # this will contain the full path between nested categories
+    # Air Quality/Particulate Matter, for example
+    path = models.CharField(max_length=512)
+
+    node_order_by = [
+        "name",
+    ]
+
+    class Meta:
+
+        constraints = [
+            models.UniqueConstraint(
+                functions.Lower("name").asc(),
+                fields=["organization", "project"],
+                include=["path"],
+                name="category_organization_project_unique_constraint",
+            )
+        ]
+
+        verbose_name = "Category"
+        verbose_plural = "Categories"
+
+
+class Metric(models.Model):
+    """
+    Represents a metric that needs to be processed and will be used by the system.
+
+    For reference, the legacy "model" that represent a metric will be in separate
+    places, according to the following list:
+
+    Metric:
+    * "metric_name": "threatened_endangered_vertebrate_species_richness"
+    * "display_name": "Threatened/Endangered Vertebrate Species Richness"
+
+    Dataset
+    * "data_units": "Number of Species"
+    * "data_provider": "FVEG 2023, California Department of Fish and Wildlife",
+    * "source": "Northern California Regional Resource Kit",
+    * "source_link": "https://wildfiretaskforce.org/northern-california-regional-resource-kit/",
+    * "reference_link": "https://rrk.sdsc.edu/norcal/p/Northern%20California%20Region%20Metric%20Dictionary18Oct23.pdf#page=42",
+
+    Treatment Goals
+    * "output_units": "Avg number of Species"
+
+    Will be removed
+    * "max_value": 6 - can be calculated from raster
+    * "min_value": 0 - can be calculated from raster
+    * "raw_data_download_path": "northern-california/bioDiverConserv/Tier2/t_e_species_richness_202304.tif" - can be calculated
+    * "raw_layer": ":bioDiverConserv_Tier2_t_e_species_richness_202304"
+    * "data_download_link": "https://data.fs.usda.gov/geodata/rastergateway/regional-resource-kit/docs/forestResil.zip"
+    * "data_year": "2021"
+    * "invert_raw": true
+    """
+
+    owner = models.ForeignKey(
+        User,
+        on_delete=models.RESTRICT,
+        related_name="owned_metrics",
+    )
+
+    project = models.ForeignKey(
+        Project,
+        on_delete=models.RESTRICT,
+        related_name="metrics",
+        null=True,
+    )
+
+    dataset = models.ForeignKey(
+        Dataset,
+        on_delete=models.RESTRICT,
+        related_name="metrics",
+    )
+
+    # a metric needs to have a category
+    # a metric can only have a category that matches
+    # the project they have
+    # do we need this?
+    category = models.ForeignKey(
+        Category,
+        related_name="metrics",
+        on_delete=models.RESTRICT,
+        help_text="category that this metric belongs to. e.g. Air Quality/Particulate Matter",
+    )
+
+    name = models.CharField(max_length=128)
+
+    display_name = models.CharField(max_length=128)
+
+    capabilities = ArrayField(
```

Comment:
not sure I understand what is this field, what else are we storing about the category besides name and display name?